source/main/ipc/getHardwareWalletChannel.ts

```diff
@@ -477,56 +480,58 @@ export const handleHardwareWalletRequests = async (
       ? utils.str_to_path(stakingPathStr)
       : null;
 
-    try {
-      deviceConnection = get(devicesMemo, [devicePath, 'AdaConnection']);
+    deviceConnection = get(devicesMemo, [devicePath, 'AdaConnection']);
```

Comment:
The whole `try/catch` block was redundant here as the only thing it did was re-throwing the error.

Reply:
Pro tip: use the `hide whitespace` GitHub feature to see the actual changes:
![image](https://user-images.githubusercontent.com/6684554/170491074-b021ab94-2d35-4100-9a48-53ed2b39c506.png)